integration-testing/build.gradle

```diff
@@ -130,3 +172,8 @@ compileTestKotlin {
 check {
     dependsOn([jvmCoreTest, debugDynamicAgentTest, mavenTest, debugAgentTest, coreAgentTest, 'smokeTest:build'])
 }
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "1.8"
```

Comment:
Isn't it enabled by default?

Reply:
Yes, 1.8 is enabled by default, but jvmTarget = "1.8" is used everywhere in the project build scripts.  Also, maybe it's better to define the version explicitly?